CHANGELOG.md

```diff
@@ -1,3 +1,9 @@
+### Release [1.3.3], 2023-01-18
+#### Bug Fix
```

Comment:
Is it worth mentioning `Ephemeral materialization` is officially supported with a know limitation with INSERT statement?

Reply:
yes, added